src/toolbox.ts

```diff
@@ -1,46 +1,136 @@
-let toolboxString = `<xml>
-        <category name="Logic" colour="%{BKY_LOGIC_HUE}">
-            <block type="controls_if"></block>
-            <block type="logic_compare"></block>
-            <block type="logic_operation"></block>
-            <block type="logic_negate"></block>
-            <block type="logic_boolean"></block>
-        </category>
-        <category name="Loops" colour="%{BKY_LOOPS_HUE}">
-            <block type="controls_repeat_ext">
-            <value name="TIMES">
-                <block type="math_number">
-                <field name="NUM">10</field>
-                </block>
-            </value>
-            </block>
-            <block type="controls_whileUntil"></block>
-        </category>
-        <category name="Math" colour="%{BKY_MATH_HUE}">
-            <block type="math_number">
-            <field name="NUM">123</field>
-            </block>
-            <block type="math_arithmetic"></block>
-            <block type="math_single"></block>
-        </category>
-        <category name="Text" colour="%{BKY_TEXTS_HUE}">
-            <block type="text"></block>
-            <block type="text_length"></block>
-            <block type="text_print"></block>
-        </category>
-        <category name="Variables" custom="VARIABLE" colour="%{BKY_VARIABLES_HUE}">
-            </category>
-
-        <sep css-container="sepDark"></sep>
-        <category name='Base' colour='#c75bc2'>
-
-        <block type='on_bot_code_start'/>
-        <block type='bot_amount_info'/>
-        <block type="bot_shutdown_stop"/>
-        <block type="set_bot_game_stream"/>
-        <block type="env"/>
-        </category>
-
-        </xml>`
+// done in a seconds with https://replit.com/@RedMan13/blockly-xml-to-json-toolbox 
```

Comment:
```suggestion
```

Reply:
We could credit it in a contributor system file later

Reply:
its litteraly just me XD

Reply:
ye ikr but we won't do credits with comments, we will do that properly

Reply:
ah which file is it? or is it not here yet

Reply:
There is none yet, we'll see that for v2 to make it automatic, np